app/controllers/declaration/BorderTransportController.scala

```diff
@@ -42,25 +42,28 @@ class BorderTransportController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with ModelCacheable {
 
-  def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
+  private val validTypes = Seq(DeclarationType.STANDARD, DeclarationType.SUPPLEMENTARY)
+
+  def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType(validTypes: _*)) { implicit request =>
```

Comment:
Perhaps its worth adding an extra method to JourneyAction  that accepts an `Iterable` so you dont have to do the `:_*` everywhere

Reply:
Yes, I was going to ask about doing something like that.   Consider it done.